### ISSUE TITLE
migrations missing NonIntPKModel

### ISSUE THREAD

**Issue by brianmay** [django-guardian/django-guardian#284]
Please run:

```
$ export DJANGO_SETTINGS_MODULE="guardian.testsettings"
$ python /usr/bin/django-admin makemigrations  testapp
  0003_auto_20141123_1837.py:
    - Create model NonIntPKModel
    - Alter field username on customuser
```

And commit the guardian/testapp/migrations/0003_auto_20141123_1837.py file.

**Comment by brianmay** [comment]
Hello @brianmay. Fancy you being interested in this project too!

Now:

```
  0004_auto_20151112_1702.py:
    - Change managers on customuser
    - Alter field email on customuser
    - Alter field groups on customuser
    - Alter field last_login on customuser
    - Alter field username on customuser
```

Will need to investigate these later. See if they are real changes.

**Comment by brianmay** [comment]
Fixed in 675f1d28389b160b394754567f1a861aaf492dc3.
